.github/workflows/image-build-on-push.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build latest images on push event
+
+on:
+  push:
+    branches: [ 'main' ]
+    paths:
+      - "**/docker/*.py"
+      - "**/docker/Dockerfile"
+      - "**/docker/ui/**"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-on-push
+  cancel-in-progress: true
+
+jobs:
+  job1:
+    uses: ./.github/workflows/reuse-get-test-matrix.yml
```

Comment:
Please verify this workflow on push.

Reply:
Yes, this is verified: https://github.com/opea-project/GenAIExamples/actions/runs/9515954706/job/26231251739
But I still need to verify if reuse-get-test-matrix works for pull_request_target.

Reply:
This has been verified in
- push: https://github.com/opea-project/GenAIExamples/actions/runs/9541582520/job/26294965584
- pull_request: https://github.com/opea-project/GenAIExamples/actions/runs/9541606106/job/26295017006
- pull_request_target: https://github.com/opea-project/GenAIExamples/actions/runs/9541606071/job/26295019580